systray.go

```diff
@@ -17,7 +17,7 @@ import (
 func onReady() {
 	systray.SetIcon(icon.Data)
 	mRebind := systray.AddMenuItem("Change Hotkey", "Choose a new hotkey")
-	mAbout := systray.AddMenuItem("About Nyrna " + VERSION, "Open changelog")
+	mAbout := systray.AddMenuItem("About Nyrna "+VERSION, "Open changelog")
```

Comment:
This doesn't look right. :smiley: Spaces around operators should be the same across the entire code, don't you think? You could as well do `mAbout:=systray...`

Reply:
The linter was collapsing this item because it was about + VERSION was considered 1 argument. Should be fixed now.

Nice catch :+1: